proxy/src/redis/notifications.rs

```diff
@@ -30,6 +30,11 @@ async fn try_connect(client: &ConnectionWithCredentialsProvider) -> anyhow::Resu
     Ok(conn)
 }
 
+#[derive(Debug, Deserialize)]
+struct NotificationHeader<'a> {
+    topic: &'a str,
+}
+
 #[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
```

Comment:
Let's leave a comment here stating that the topic match below needs to be updated.

Reply:
Or maybe do it in another PR to avoid re-running all the tests.